freetextresponse/private/view.js

```diff
@@ -4,46 +4,125 @@ function FreeTextResponseView(runtime, element) {
     var $ = window.jQuery;
     var $element = $(element);
     var buttonSubmit = $element.find('.check.Submit');
-    var textareaParent = $element.find('.student_answer').parent();
+    var textareaParent = $element.find('.textarea').parent();
     var usedAttemptsFeedback = $element.find('.action .used-attempts-feedback');
     var problemProgress = $element.find('.problem-progress');
     var submitParent = $element.find('.Submit').parent();
-    var wordCountError = $element.find('.word-count-error');
     var submissionReceivedMessage = $element.find('.submission-received');
     var url = runtime.handlerUrl(element, 'submit');
     var userInputClass = 'user-input';
+    var buttonSave = $element.find('.save');
+    var saveParent = $element.find('.save').parent();
+    var urlSave = runtime.handlerUrl(element, 'save_reponse');
+    var userAlertMessage = $element.find('.user_alert');
+    var statusIndicator = $element.find('.status');
+    var textarea = $element.find('.textarea');
+    var textareaParentParent = textareaParent.parent();
+    
+    function _setBorderColor(indicator_class) {
+        if ( 'correct' == indicator_class ) {
+            textarea.css('border', 'solid #1e9348');
+        } else if ( 'incorrect' == indicator_class ) {
+            textarea.css('border', 'solid #b20610');
+        } else {
+            textarea.css('border', 'solid #e4e4e4');
```

Comment:
The textarea css had to be inline html?   That was the only way I could get the border to change color.  I'll clean up everything else and we can look at this in more detail.